docs/intro/1_whats_in_the_box.md

```diff
@@ -48,6 +51,7 @@ and provide the following **technology stack**:
 - **Cordova** and it's **CLI** for accessing phone APIs and building the app - http://cordova.apache.org/
   - **ngCordova** connecting Angular with Cordova http://ngcordova.com/
 - **SASS** as the CSS preprocessor of choice - http://sass-lang.com/
+- **JADE** as the HTML template engine of choice - http://jade-lang.com/
```

Comment:
Would like `**JADE** as an opt-in HTML template engine` better, since I rather see it as a neat drop in feature than a dev-stack recommendation.